### DISCUSSION TITLE
Question: Does `river` work with `cockroachdb`?

### DISCUSSION THREAD

**Discussion by binaek** [riverqueue/river#442] [Q&A] [answered]
CockroachDB claims to be a drop in replacement for postgresql. Is `riverqueue` compatible with cockroachdb?

**Comment by bgentry** [comment, marked as answer]
No, see https://github.com/riverqueue/river/issues/104#issuecomment-1848856246 and #56. Cockroach is definitely not a drop-in replacement even though it uses the Postgres wire protocol.

**Comment by binaek** [comment]
Makes sense. Thanks for the quick response.
